fe1-web/src/core/components/Input.tsx

```diff
@@ -27,13 +27,21 @@ export const inputStyleSheet = StyleSheet.create({
   disabled: {
     color: Color.gray,
   },
+  monospaced: {
+    ...Typography.code,
+  },
 });
 
 const Input = (props: IPropTypes) => {
-  const { value, placeholder, onChange, onFocus, onBlur, enabled, negative, testID } = props;
+  const { value, placeholder, onChange, onFocus, onBlur, enabled, negative, testID, isMonospaced } =
+    props;
 
   const inputStyles = [Typography.paragraph, inputStyleSheet.input];
 
+  if (isMonospaced) {
+    inputStyles.push(inputStyleSheet.monospaced);
```

Comment:
You should be able to push Typography.code directly here :)

Reply:
Oh, this is good to know. 
This is indeed better, as it allows having more readable code. The monospaced font is a single attribute that only should be changed globally and not at the granularity of the component.